File: 100-Days-Series/Day 29/CountWord.java
class CountWord{
    // Main Metod 
    public static void main(String [] args){
        // Let's Start here
        String sentence;
        java.util.Scanner sc=new java.util.Scanner(System.in);

        System.out.print("Enter a sentence : ");
        sentence=sc.nextLine();

        if(sentence.trim().isEmpty()){
            System.out.println("The number of words in the sentence is : 0");
        }else{
            String arr[]=sentence.trim().split("\\s+");
            System.out.println("The number of words in the sentence is : "+arr.length);
        }
    }
}

File: 100-Days-Series/Day 29/CountWord.js

// Let's Start here
let sentence="Full Stack Developer Veer";

if(sentence.trim()===""){
    console.log("The number of words in the sentence is : 0");
}
else{
    let words=sentence.trim().split(/\s+/);
    console.log("The number of words in the sentence is :"+words.length);
}